### ISSUE TITLE
Expand Tab container child element padding for improved UX on tab navigation

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#6775]
The sheet-tabs class currently lacks buffer space on its child elements; as a result the individual `<a>` elements in the flex row have a small profile for clicking (limited to the text of the link) which can be difficult for low mobility and low sight users. 

I'm not certain that a padding adjustment is the right way forward here; if feasible, perhaps there's a media query or something here which could be used to determine the number of child elements in the tabs container and set the min-width of each `<a>` element to an equal division of that.

**Comment by Fyorl** [comment]
It's quite simple to give every element in the tab row an equal share of the row's width by setting `flex-basis: 100%`. The problem is that it just doesn't look very good:

Before:
![Image](https://user-images.githubusercontent.com/207433/180830507-861a23c5-fdd5-4400-a267-c360dfc25e62.png)

After:
![Image](https://user-images.githubusercontent.com/207433/180830367-832695a2-8bf6-49b9-ba6e-a564bee5b870.png)

In the latter case, you end up with a variable amount of space between the elements in the tab. For example, the gap between Vision and Light is quite large compared to the others.

It looks more acceptable when you give the tabs borders to clearly demarcate their boundaries, but that's a much more significant change than is being proposed in this issue:
![Image](https://user-images.githubusercontent.com/207433/180832072-9a058079-3d8f-4768-8a0e-e84942e1a106.png)

**Comment by anathemamask** [comment]
I think direct visibility of clear boundaries is probably not necessary as long as clicking "near" the `<a>` works--the user who requested this was mostly running into issues with ability to click on the text specifically, compared to clicking on the header/tab that contains the text.

**Comment by aaclayton** [comment]
I think what is desired here is that, instead of placing the padding on the `.tabs` parent with un-padded child `.item` we would place padding on the children to expand their "hit area".

i.e. remove padding from `.sheet nav.sheet-tabs` and add `padding: 8px` to `.sheet .tabs.item`
